{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specifications/model_spec.in

```diff
@@ -7,7 +7,7 @@ components:
             - DisabilityObserver()
             - MortalityObserver()
 
-    {{cookiecutter.package_name}}.components:
+    # {{cookiecutter.package_name}}.components:
```

Comment:
Add a comment about why this is commented out? It looks a bit strange as is, though I know why it's being done this way.